src/approaches/interactive_learning_approach.py

```diff
@@ -38,6 +38,9 @@ def _get_current_predicates(self) -> Set[Predicate]:
     ######################## Semi-supervised learning #########################
 
     def learn_from_offline_dataset(self, dataset: Dataset) -> None:
+        # Special case: empty offline dataset. Has no annotations.
+        if not dataset.trajectories:
+            return
```

Comment:
what breaks if you don't do this?

Reply:
oh, is it that dataset.annotations will be None?

Reply:
yeah (see the comment)

Reply:
oh i see, it's not None it'll actually just crash upon access. nvm